### PR TITLE
Added Wasted LOD - Cliffs of DC

### PR DESCRIPTION
**Pull request by Superfly-Johnson**
I feel like adding a "Requires TTW" message would be a good idea, this mod and several TTW mods need it.

### CODE REVIEW

**Comment by MacSplody** [comment]
> I feel like adding a "Requires TTW" message would be a good idea, this mod and several TTW mods need it.

Does it have TTW as a master file?
If it does then the user will already recieve an error.
If not then yeah a `req` can be used.

**Comment by Superfly-Johnson** [comment]
> > I feel like adding a "Requires TTW" message would be a good idea, this mod and several TTW mods need it.
> 
> Does it have TTW as a master file? If it does then the user will already recieve an error. If not then yeah a `req` can be used.

It doesn't use TTW as a master file no, LOD modules usually have no masters at all. I'll add a req a bit later

**Comment by MacSplody** [comment]
You'll need to rebase the changes, as it conflicts with the changes that were just merged.

Edit:
When you rebase you have some conflicts that need to be resolved.
The conflicts are usually wrapped with
`>>>>>>`
and
`<<<<<`

**Review by MacSplody — APPROVED** [review]

File: masterlist.yaml [hunk 1]
```diff
@@ -5392,6 +5392,16 @@ plugins:
       - crc: 0x5BD687F8
         util: 'FNVEdit v4.1.4m  EXTREMELY EXPERIMENTAL'
 
+  - name: 'Wasted LOD rocks.esp'
+    url:
+    - link: 'https://www.nexusmods.com/newvegas/mods/79734/'
+      name: 'Wasted LOD - Cliffs of DC'
+    group: *lodGroup
+    req: *TTWv320
```
Comment (MacSplody) [inline masterlist.yaml]:
```suggestion
    req: [ *TTWv320 ]
```

**Pull request merged**